Tasks/Maven/maventask.ts

```diff
@@ -1,4 +1,5 @@
 /// <reference path="../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../definitions/codecoveragefactory.d.ts" />
```

Comment:
i will do once all reviews are done and final down merge from source branch is completed.